src/platform_impl/linux/x11/window.rs

```diff
@@ -410,11 +420,11 @@ impl UnownedWindow {
                 unsafe {
                     // XSetInputFocus generates an error if the window is not visible, so we wait
                     // until we receive VisibilityNotify.
-                    let mut event = mem::uninitialized();
```

Comment:
Why not `mem::zeroed()` and be done with it?

Reply:
`XEvent` has no `Default` impl. It's a union type with a size of 192 bytes.